src/util/util_watchdog.c

```diff
@@ -78,9 +75,7 @@ static void watchdog_handler(int sig)
 
```

Comment:
In this implementation, if the timeshift is detected we should continue and increase the watchdog tick to avoid this scenario:
1. SSSD is not responding
2. Timeshift is detected and write to the pipe was successful
3. Watchdog is not restarted (because we didn't get to the fd handler since sssd is not processing the loop)
4. Process is not killed because we always detect the timeshift.